test/namespace_admin_test.py

```diff
@@ -159,6 +171,11 @@ def cleanup_kubeconfig(self):
         except Exception as e:
             raise K8sError(f"Could not write to KUBECONFIG: {KUBECONFIG}") from e
 
+    def bob_runs_scenario_tests(self):
+        assert self.this_is_the_current_context(self.bob_context)
+        super().run_test()
```

Comment:
![Screenshot 2024-10-09 at 10 15 57 PM](https://github.com/user-attachments/assets/07d42b16-8baa-4649-9625-c45b258a68c7)

Reply:
Bob re-runs the helm commands, and if I understand it correctly, this is why he needs the `patch` permission. I actually want to keep this as part of the test because it seems like a good thing to make sure pod-manager service accounts like bob have access to it.